github/codeowners.py

```diff
@@ -81,73 +111,173 @@ def filter_codeowners_entries(
         yield from line.split()[1:]
 
 
-def _first(
-    iterable: typing.Iterable,
-):
+def resolve_team_members(
+    team_name: str,
+    org: github3.orgs.Organization,
+    absent_ok: bool=False,
+) -> typing.Generator[github3.users.ShortUser, None, None]:
+    '''
+    Returns generator yielding members from team.
+    Absent teams passed silently with `absent_ok`.
+    Subteams are resolved as well.
```

Comment:
a more common (and accurate) wording would be to mention nested teams being resolved recursively; also, since this is the core of this function's logic, I think it should be included int the first sentence, rather than being added as an additional hint after documenting the semantics of the `absent_ok` parameter.

I think we also should allow `Teamname` as input as well (-> typehint)